### PR TITLE
Feature/add businesses

### PR DESCRIPTION
**Pull request by elijahmandeville**
Created migration table and requisite seed file. Added all get, put, post, and delete endpoints for businesses table. Added a get endpoint to profile router to use /profiles url.

### CODE REVIEW

File: data/seeds/003_businesses.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    .then(function () {
+      // Inserts seed entries
+      return knex('businesses').insert([
+        {
+          user_id: '00ulthapbErVUwVJy4x6',
+          email: 'groomerexpress@yahoo.com',
+          password: 'groomer',
+          name: 'Groomer Express',
+          profile_photo:
+            'https://imageproxy.themaven.net//https%3A%2F%2Fedm.com%2F.image%2FMTc0NjYyNjI0ODg4MjM1Mzg2%2Fscience-of-earworms-explain-why-never-gonna-give-you-up-is-stuck-in-our-heads-30-years-later.png',
```
Comment (Iinh) [inline data/seeds/003_businesses.js:14]: Don't mind me being nitpicky here: I think we should shorten long urls so the returned data on the client side looks cleaner. We could use [Bitly](https://bitly.com/) for this 😄 
```suggestion
            'https://bit.ly/2TWHF9n',
```

**Comment by Iinh** [comment]
@elijahmandeville Everything looks great! Code was easy to read and follow, I made one suggestion inline. Other than that I think we're good to merge!

**Review by Samuel-Torres — APPROVED** [review]
Looks like this file is failing due to test coverage. I know we're not currently writing tests but if you'd like to give it a swing might be useful to go over again! :)

File: data/seeds/003_businesses.js (hunk 1 above)
Comment (elijahmandeville) [reply data/seeds/003_businesses.js:14]: That's a great idea! I will change that and use bitly in the future.

**Pull request merged**